### PR TITLE
Citra wiki optimization

### PR DESCRIPTION
**Pull request by robzombie91**
Just thought I would go through most of the entries that are old and try to clean them up. Didn't touch any virtual console or e-shop entries because I just didn't feel like trying to find their title-ids or release dates because trying to find the normal releases was hard enough. aka too lazy

P.S. If the spaces in-between some tags is too much I can go back and edit them out.

### CODE REVIEW

File: games/etrian-odyssey-v-beyond-the-myth/game.dat [hunk 1]
```diff
@@ -13,6 +13,11 @@ title = "00040000001C5300"
 region = "eur"
 release_date = "2017-11-03"
 
+[[ releases ]]
+title = "000400000018D000"
+region = "jpn"
+release_date = "2016-08-04"
```
Comment (Leo626) [inline games/etrian-odyssey-v-beyond-the-myth/game.dat]: This is a duplicate jpn release.

File: games/story-of-seasons-trio-of-towns/game.dat [hunk 2]
```diff
@@ -9,6 +9,17 @@ title = "000400000019F500"
...
+title = "00040000001B5500"
+region = "eur"
+release_date = "2017-10-13"
+
+[[ releases ]]
+title = "0004000000073700"
+region = "jpn"
+release_date = "2016-05-23"
+
+
```
Comment (BreadFish64) [inline games/story-of-seasons-trio-of-towns/game.dat]: remove newline

File: games/story-of-seasons-trio-of-towns/game.dat [hunk 3]
```diff
@@ -9,6 +9,17 @@ title = "000400000019F500"
 region = "usa"
 release_date = "2017-02-28"
 
+[[ releases ]]
+title = "00040000001B5500"
+region = "eur"
+release_date = "2017-10-13"
+
+[[ releases ]]
+title = "0004000000073700"
```
Comment (BreadFish64) [inline games/story-of-seasons-trio-of-towns/game.dat]: 0004000000185300
the one you put is harvest moon 3d

File: games/super-mario-3d-land/game.dat [hunk 4]
```diff
@@ -1,5 +1,5 @@
 title = "Super Mario 3D Land"
-description = "Super Mario 3D Land is a platform game in the Super Mario series developed by Nintendo EAD Tokyo, with development assistance from Brownie Brown, for the Nintendo 3DS. The game was the first in the Mario franchise to be released for the system and was released worldwide in November 2011."
```
Comment (BreadFish64) [inline games/super-mario-3d-land/game.dat]: keep the part about being the first mario game for 3ds, just remove the release date stuff

File: games/terraria/game.dat [hunk 5]
```diff
@@ -3,9 +3,19 @@ description = "Explore vast fantasy worlds by excavating down to the depths of t
 needs_system_files = true
 needs_shared_font = false
 
+[[releases]]
+title = "000400000016A900"
+region = "all"
```
Comment (BreadFish64) [inline games/terraria/game.dat]: region = "usa"

File: games/terraria/game.dat [hunk 6]
```diff
@@ -3,9 +3,19 @@ description = "Explore vast fantasy worlds by excavating down to the depths of t
...
+release_date = "2015-12-10"
+
 [[releases]]
 title = "000400000016A600"
-region = "eur"
+region = "all"
```
Comment (BreadFish64) [inline games/terraria/game.dat]: region = "eur"

File: games/terraria/game.dat [hunk 7]
```diff
@@ -3,9 +3,19 @@ description = "Explore vast fantasy worlds by excavating down to the depths of t
...
+release_date = "2015-12-10"
+
+[[releases]]
+title = "0004000000193500"
+region = "all"
```
Comment (BreadFish64) [inline games/terraria/game.dat]: region = "jpn"

**Comment by BreadFish64** [comment]
For future reference, it'd be easier to use git or the github desktop app so you can clone the repository to change things locally and upload them in one commit... not 250+ :P

File: games/pokemon-mystery-dungeon-gates-to-infinity/game.dat [hunk 8]
```diff
@@ -14,6 +14,11 @@ title = "00040000000BA900"
 region = "eur"
 release_date = "2013-05-17"
 
+[[ releases ]]
+title = "0004000000149B00"
```
Comment (Leo626) [inline games/pokemon-mystery-dungeon-gates-to-infinity/game.dat]: Wrong title ID. Should be `0004000000095000`.

File: games/story-of-seasons-trio-of-towns/game.dat (hunk 3 above)
Comment (Leo626) [inline games/story-of-seasons-trio-of-towns/game.dat]: The correct title ID should be `0004000000185300`.

File: games/zero-escape-virtues-last-reward/game.dat [hunk 9]
```diff
@@ -4,9 +4,19 @@ needs_system_files = false
...
 region = "usa"
+release_date = "2012-10-23"
+
+[[ releases ]]
+title = "00040000000AEB00"
+region = "eur"
+release_date = "2012-11-23"
+
+[[ releases ]]
+title = "0004000000180F00"
```
Comment (Leo626) [inline games/zero-escape-virtues-last-reward/game.dat]: Wrong title ID. Should be `000400000005DE00`.

**Comment by Leo626** [comment]
* Add Corpse Party eur release. Title ID is 0004000000198300.

* Pokemon Rumble World's Retail and eShop release have different title IDs.

0004000000185A00 (Retail) - usa (2016-04-29), eur (2016-01-22), jpn (2015,11-19)
0004000000164600 (eShop) - all (2015-04-08)

* You changed the boxart for Pokemon XY to X's but the icon is still Y's. For consistency the icon should be changed to X's.

**Comment by robzombie91** [comment]
it still failed even after commiting a new box-art

**Pull request closed without merging**